src/main/java/com/fakng/fakngagrgtr/parser/apple/LocationDTO.java

```diff
@@ -0,0 +1,24 @@
+package com.fakng.fakngagrgtr.parser.apple;
+
+import lombok.Data;
+
+@Data
+public class LocationDTO {
+
+    private String city;
+
+    private Integer level;
```

Comment:
I think we use only "city" and "countryID" fields here. Maybe some other one, but writing the rest is redundant, if we ignore those. The same for other DTO classes. Please, get rid of things that are unused

Reply:
Yes, good. Then, if anything, it will be possible to painlessly add them